Gopkg.toml

```diff
@@ -105,3 +105,7 @@ required = [
 [prune]
   go-tests = true
   unused-packages = true
+
+  [[prune.project]]
+    name = "github.com/ligato/cn-infra"
+    unused-packages = false
```

Comment:
There might be packages that are currently not used, but may be used later and to prevent having to re-run `dep ensure` each time any of the dependency on cn-infra package is added/removed, we can just keep it all.